### ISSUE TITLE
490/880 conversion

### ISSUE THREAD

**Issue by jodiw01** [lcnetdev/marc2bibframe2#243]
Using LCCN 2022470726 as the example.

The paired 490-880 fields are placed in a single series Hub resource, but the 880 field is converted again to create a second series Hub resource. This conversion should stop. Per the specs, just the 490-880 combo resource is needed.

Needed:

```
<bflc:relationship>
<bflc:Relationship>
<bflc:relation>
<bflc:Relation rdf:about="http://id.loc.gov/ontologies/bibframe/hasSeries">
<rdfs:label>Has Series</rdfs:label>
</bflc:Relation>
</bflc:relation>
<bf:relatedTo>
<bf:Series rdf:about="http://id.loc.gov/resources/REPLACE/22987419#Hub490-28">
<rdf:type rdf:resource="http://id.loc.gov/ontologies/bflc/Uncontrolled"/>
<bf:status>
<bf:Status rdf:about="http://id.loc.gov/vocabulary/mstatus/t">
<rdfs:label>transcribed</rdfs:label>
</bf:Status>
</bf:status>
<bf:title>
<bf:Title>
<bf:mainTitle>Serii︠a︡ "Istorii︠a︡ ta polityka"</bf:mainTitle>
<bf:mainTitle xml:lang="uk-cyrl">Серія "Історія та політика"</bf:mainTitle>
</bf:Title>
</bf:title>
</bf:Series>
</bf:relatedTo>
</bflc:Relationship>
</bflc:relationship>

```

Currently created but not needed:

```
<bflc:relationship>
<bflc:Relationship>
<bflc:relation>
<bflc:Relation rdf:about="http://id.loc.gov/ontologies/bibframe/hasSeries">
<rdfs:label>Has Series</rdfs:label>
</bflc:Relation>
</bflc:relation>
<bf:relatedTo>
<bf:Series rdf:about="http://id.loc.gov/resources/REPLACE/22987419#Hub880-29">
<rdf:type rdf:resource="http://id.loc.gov/ontologies/bflc/Uncontrolled"/>
<bf:status>
<bf:Status rdf:about="http://id.loc.gov/vocabulary/mstatus/t">
<rdfs:label>transcribed</rdfs:label>
</bf:Status>
</bf:status>
<bf:title>
<bf:Title>
<bf:mainTitle>Серія "Історія та політика"</bf:mainTitle>
</bf:Title>
</bf:title>
</bf:Series>
</bf:relatedTo>
</bflc:Relationship>
</bflc:relationship>
```

**Comment by jodiw01** [comment]
related to https://github.com/lcnetdev/bibframe2marc/issues/28

**Comment by jodiw01** [comment]
fixed in marc2bibframe v2.7
